decidim-core/app/assets/stylesheets/decidim/modules/_metrics.scss

```diff
@@ -1,16 +0,0 @@
-.metric-charts{
```

Comment:
Is this supposed to be removed?

Reply:
Yes. This was a temporal CSS to modify UX design until we finish all other metrics.